### ISSUE TITLE
Bug: id is not parsed in <use/> tag

### ISSUE THREAD

**Issue by ngaurav** [dnfield/flutter_svg#273]
`<use id='g19-49' transform='scale(0.5)' xlink:href='#g4-49'/>`
`<use x='76.3632' xlink:href='#g19-49' y='-5.20292'/>`

While parsing the second line an error is thrown with this message:
Expected to find Drawable with id url(#g19-49)

The full SVG code is below:
```xml
<?xml version='1.0' encoding='UTF-8'?>
<!-- This file was generated by dvisvgm 2.1.3 -->
<svg height='22.2426pt' version='1.1' viewBox='0.198642 -19.8213 104.085 22.2426' width='104.085pt' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'>
<defs>
<path d='M2.00098 8.11826L2.80137 7.44364L4.74518 12.3718H5.49983L11.1026 0H11.8687V-0.68605H10.6681L5.44266 10.8396L3.54459 6.01437L1.60078 7.63802L2.00098 8.11826Z' id='g1-112'/>
<path d='M1.88664 10.3937L2.68703 9.71904L4.74518 16.7282H5.49983L11.1483 0H11.8687V-0.68605H10.6223L5.44266 14.67L3.54459 8.1983L1.48644 9.91342L1.88664 10.3937Z' id='g1-113'/>
<path d='M1.8066 12.6233L2.60699 11.9601L4.74518 21.0846H5.49983L11.1712 0H11.8687V-0.68605H10.5995L5.44266 18.5005L3.54459 10.3708L1.41784 12.1545L1.8066 12.6233Z' id='g1-114'/>
<path d='M1.65795 7.04345L2.45835 6.36883L3.88762 10.1993H4.64227L9.19307 0H9.94773V-0.68605H8.7357L4.59654 8.60993L3.22444 4.92813L1.26919 6.57465L1.65795 7.04345Z' id='g3-112'/>
<path d='M1.03479 -2.75563L0.188664 -2.56697V-2.42404L0.657465 -2.33829L0.182947 0H0.668899L1.02336 -1.93237C1.3721 -2.25253 1.67511 -2.40118 1.86949 -2.40118C2.08674 -2.40118 2.17821 -2.29255 2.17821 -2.041C2.17821 -1.96668 2.16677 -1.8409 2.12676 -1.67511L1.88092 -0.617445C1.85234 -0.491669 1.83518 -0.383045 1.83518 -0.308723C1.83518 -0.0800392 1.97239 0.0343025 2.25253 0.0343025C2.41261 0.0343025 2.64129 -0.0228683 2.93858 -0.13721V-0.297288C2.80709 -0.262986 2.70418 -0.245835 2.62414 -0.245835C2.43548 -0.245835 2.32685 -0.343025 2.32685 -0.525972C2.32685 -0.611728 2.35544 -0.737504 2.38974 -0.903299L2.58412 -1.78945C2.62986 -1.99526 2.66416 -2.15534 2.66416 -2.29827C2.66416 -2.5784 2.48121 -2.75563 2.18393 -2.75563C1.88092 -2.75563 1.52646 -2.56697 1.07481 -2.20679L1.17772 -2.75563H1.03479Z' id='g7-110'/>
<path d='M0.966187 -2.75563L0.154361 -2.58412V-2.44119L0.600294 -2.38402L0 0.943319L-0.0571708 1.00621L-0.394479 1.07481V1.24632H0.977621V1.07481L0.434498 0.977621L0.623162 -0.0857563C0.766089 -0.0171513 0.914733 0.0228683 1.06338 0.0228683C1.45214 0.0228683 1.85805 -0.165795 2.14962 -0.485952C2.4755 -0.846128 2.63558 -1.26348 2.63558 -1.76086C2.63558 -2.40689 2.36116 -2.75563 1.90951 -2.75563C1.66367 -2.75563 1.37782 -2.64129 1.04623 -2.40689L1.11483 -2.75563H0.966187ZM0.680333 -0.383045L1.00621 -2.15534C1.26348 -2.35544 1.47501 -2.45835 1.64652 -2.45835C1.95524 -2.45835 2.16106 -2.20108 2.16106 -1.74943C2.16106 -1.29206 2.00098 -0.828977 1.81232 -0.577425C1.63509 -0.343025 1.42355 -0.228683 1.172 -0.228683C1.01192 -0.228683 0.851846 -0.280137 0.680333 -0.383045Z' id='g7-112'/>
<path d='M3.55603 -7.80954L1.07481 -7.06632V-6.58608L2.86998 -6.90624V-0.754655L2.73277 -0.560274L1.41784 -0.491669V0H5.24828V-0.480235L4.01339 -0.537406L3.87618 -0.708918V-7.80954H3.55603Z' id='g4-49'/>
<path d='M1.61222 -0.903299C4.13917 -2.91571 5.5227 -4.20777 5.5227 -5.64848C5.5227 -6.94054 4.6194 -7.80954 3.1444 -7.80954C1.72656 -7.80954 0.720353 -6.99771 0.720353 -6.19732C0.720353 -5.80856 0.96047 -5.55701 1.36067 -5.55701C1.54361 -5.55701 1.71513 -5.61418 1.8752 -5.72852V-6.95197C2.21823 -7.14635 2.54982 -7.22639 2.89284 -7.22639C3.84188 -7.22639 4.41359 -6.58608 4.41359 -5.49983C4.41359 -4.00196 2.98432 -2.75563 0.480235 -0.720353V0H5.77425V-1.8752H5.25972L5.0539 -0.903299H1.61222Z' id='g4-50'/>
<path d='M4.20777 -7.74093L0.308723 -2.96145V-2.21823H3.99052V0.423064H4.87096V-2.21823H6.16302V-2.96145H4.87096V-7.74093H4.20777ZM3.99052 -6.49461V-2.96145H1.16629L3.99052 -6.49461Z' id='g4-52'/>
<path d='M1.15485 -7.67233V-3.89905C1.4064 -3.94479 1.73799 -3.96766 2.16106 -3.96766C3.75041 -3.96766 4.53936 -3.43025 4.53936 -2.25253C4.53936 -1.18915 3.80758 -0.423064 2.73277 -0.423064C2.48121 -0.423064 2.21823 -0.468801 1.96668 -0.560274L1.50931 -1.54361C1.39497 -1.56648 1.3035 -1.57792 1.23489 -1.57792C0.846128 -1.57792 0.571708 -1.32636 0.571708 -0.971904C0.571708 -0.423064 1.36067 0.102908 2.53839 0.102908C4.41359 0.102908 5.59131 -1.00621 5.59131 -2.48121C5.59131 -3.78471 4.6194 -4.67657 3.08723 -4.67657C2.72133 -4.67657 2.3097 -4.6194 1.86377 -4.49363V-6.78046H5.24828V-7.67233H1.15485Z' id='g4-53'/>
<use id='g19-49' transform='scale(0.5)' xlink:href='#g4-49'/>
<use id='g19-50' transform='scale(0.5)' xlink:href='#g4-50'/>
<use id='g19-52' transform='scale(0.5)' xlink:href='#g4-52'/>
<use id='g19-53' transform='scale(0.5)' xlink:href='#g4-53'/>
<use id='g11-110' transform='scale(1.33333)' xlink:href='#g7-110'/>
<use id='g15-110' transform='scale(2)' xlink:href='#g7-110'/>
<path d='M0.806109 -1.64652C0.806109 -3.213 1.26348 -3.99052 1.92094 -4.10487V-4.36785C0.971904 -4.25351 0.320157 -3.37308 0.320157 -1.64652S0.971904 0.96047 1.92094 1.07481V0.811826C1.26348 0.697484 0.806109 -0.0800392 0.806109 -1.64652Z' id='g18-40'/>
<path d='M1.43499 -1.64652C1.43499 -0.0800392 0.977621 0.697484 0.320157 0.811826V1.07481C1.26919 0.96047 1.92094 0.0800392 1.92094 -1.64652S1.26919 -4.25351 0.320157 -4.36785V-4.10487C0.977621 -3.99052 1.43499 -3.213 1.43499 -1.64652Z' id='g18-41'/>
<path d='M1.90951 -1.47501V0.114342H2.25253V-1.47501H3.84188V-1.81803H2.25253V-3.40738H1.90951V-1.81803H0.320157V-1.47501H0.491669H1.90951Z' id='g18-43'/>
<path d='M0.320157 -2.09245H4.16204V-2.43548H0.320157V-2.09245ZM0.320157 -0.857563H4.16204V-1.20059H0.320157V-0.857563Z' id='g18-61'/>
<use id='g21-40' transform='scale(1.33333)' xlink:href='#g18-40'/>
<use id='g21-41' transform='scale(1.33333)' xlink:href='#g18-41'/>
<use id='g24-61' transform='scale(2)' xlink:href='#g18-61'/>
<use id='g25-50' xlink:href='#g4-50'/>
</defs>
<g id='page1'>
<use x='3.47974' xlink:href='#g7-112' y='-11.8856'/>
<use x='0.198642' xlink:href='#g1-114' y='-19.1327'/>
<rect height='0.688626' width='71.7436' x='11.8134' y='-19.8213'/>
<use x='15.0062' xlink:href='#g19-52' y='-10.4085'/>
<use x='18.3403' xlink:href='#g7-110' y='-10.4085'/>
<use x='15.1453' xlink:href='#g1-113' y='-15.5519'/>
<rect height='0.688626' width='56.797' x='26.76' y='-16.2405'/>
<use x='29.9528' xlink:href='#g19-50' y='-8.93141'/>
<use x='33.2869' xlink:href='#g7-110' y='-8.93141'/>
<use x='30.0919' xlink:href='#g1-112' y='-12.5793'/>
<rect height='0.688626' width='41.8504' x='41.7066' y='-13.2679'/>
<use x='45.0429' xlink:href='#g7-110' y='-7.45432'/>
<use x='41.8479' xlink:href='#g3-112' y='-10.0945'/>
<rect height='0.688626' width='32.034' x='51.523' y='-10.7832'/>
<use x='51.81' xlink:href='#g15-110' y='0'/>
<use x='58.1912' xlink:href='#g21-40' y='-3.12175'/>
<use x='61.3895' xlink:href='#g11-110' y='-3.12175'/>
<use x='65.6436' xlink:href='#g19-53' y='-5.20292'/>
<use x='68.9777' xlink:href='#g7-110' y='-5.20292'/>
<use x='72.1683' xlink:href='#g18-43' y='-5.20292'/>
<use x='76.3632' xlink:href='#g19-49' y='-5.20292'/>
<use x='80.0519' xlink:href='#g21-41' y='-3.12175'/>
<use x='86.2254' xlink:href='#g24-61' y='0'/>
<use x='97.9262' xlink:href='#g25-50' y='0'/>
</g>
</svg>
```

**Comment by dnfield** [comment]
This is easy to fix, but this SVG will still have problems because of #164

**Comment by dnfield** [comment]
Both issues will be fixed by #276
